### PR TITLE
Make the homepage  

### PR DESCRIPTION
**Pull request by mattbowen-usds**
This is like half-prototype quality --- the query for the initiatives works, but I'd like to make more of it user-configurable in the next iteration.

### CODE REVIEW

File: tech-far-hub/gatsby-config.ts [hunk 1]
```diff
@@ -25,7 +25,15 @@ const config: GatsbyConfig = {
       resolve: `gatsby-plugin-mdx`,
       options: {
         extensions: [`.mdx`, `.md`],
-        gatsbyRemarkPlugins: [`gatsby-remark-autolink-headers`],
+        gatsbyRemarkPlugins: [
```
Comment (vim-usds) [inline tech-far-hub/gatsby-config.ts:28]: they seem to have a plugin for everything!

File: tech-far-hub/src/components/initiative.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+// TODO: Add media alt text
+interface IInitiativeProps {
+  children: React.ReactNode;
+  heading: React.ReactNode | string;
+  media: any;
+  destination: string;
+}
+
+export const Initiative = ({ children, heading, media, destination }: IInitiativeProps): JSX.Element => {
+  return (
```
Comment (vim-usds) [inline tech-far-hub/src/components/initiative.tsx:14]: super clean component!

File: tech-far-hub/src/components/tfh.scss [hunk 3]
```diff
@@ -1,3 +1,78 @@
+/*
+  Hide compile warnings and notifications, make want to set to true in dev
+*/
+$theme-show-compile-warnings: false;
```
Comment (vim-usds) [inline tech-far-hub/src/components/tfh.scss:4]: Nice on getting USWDS integrated! Is this working as expected?

File: tech-far-hub/src/pages/index.tsx [hunk 4]
```diff
@@ -1,15 +1,136 @@
...
+          {initiatives}
+        </Grid>
+      </Grid>
     </Layout>
   );
 };
 
 export default IndexPage;
 
 export const Head: HeadFC = () => <title>TechFAR Hub</title>;
```
Comment (vim-usds) [inline tech-far-hub/src/pages/index.tsx:109]: I'm still getting used to the new Head API

**Review by vim-usds — APPROVED** [review]
Nice work!

File: tech-far-hub/src/pages/index.tsx (hunk 4 above)
Comment (mattbowen-usds) [reply tech-far-hub/src/pages/index.tsx:109]: I clearly am not really using it yet, but I need to sooooon

**Pull request merged**